testing/features/support/components/footer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module DesignSystem
+  class Footer < ::Base
+    set_url "/iframe.html?id=3-components--footer&viewMode=story"
+
+    sections :category_titles, ".cads-grid-col-md-3 > h2" do
+      elements :links, :xpath, "./following-sibling::ul[1]/li/a"
+    end
+
+    section :initial_form, ".cads-footer" do
+      element :website_feedback, ".cads-website-feedback a"
+      element :logo, ".cads-logo"
+      element :copyright, ".cads-footer__company-info div p:nth-of-type(1)"
```

Comment:
Don't need to worry about this now, but we'll be doing a bit of design review work on the footer so can hopefully improve some of these selectors at that point.